terraform/mgmt/jenkins.tf

```diff
@@ -14,7 +14,7 @@ data "aws_ami" "jenkins" {
 
   filter {
     name = "name"
-    values = ["jenkins *"]
+    values = ["jenkins*"]
```

Comment:
Out of curiosity, why this change?

Reply:
In my case, when I build AMI’s, I use dashes as a separator instead of spaces - like “jenkins-121117-01”. This filter won’t find them. It’s a nit.